charts/mirrorbits-parent/templates/persistentvolumesecret.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if and .Values.storage.persistentVolume.azureFile.storageAccountName .Values.storage.persistentVolume.azureFile.storageAccountKey -}}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "mirrorbits-parent.pv-secretname" . }}
+  labels:
+{{ include "mirrorbits-parents.labels" . | nindent 4 }}
+type: Opaque
+data:
+  azurestorageaccountname: {{ .Values.storage.persistentVolume.azureFile.storageAccountName | b64enc }}
+  azurestorageaccountkey: {{ .Values.storage.persistentVolume.azureFile.storageAccountKey | b64enc }}
```

Comment:
No camelCase for the data field names?

Reply:
https://learn.microsoft.com/en-us/azure/aks/azure-csi-files-storage-provision#create-a-kubernetes-secret : nope (and we don't have for the current production secrets either).

As much as CamelCase convention makes sens for values file, in templates we must stick to either Kubernetes or Cloud providers syntax.